python/cuml/common/input_utils.py

```diff
@@ -298,7 +377,8 @@ def convert_dtype(X, to_dtype=np.float32, legacy=True):
                                 "in data loss.")
             return X_m
 
-    elif isinstance(X, cudf.Series) or isinstance(X, cudf.DataFrame):
+    elif isinstance(X, cudf.Series) or isinstance(X, cudf.DataFrame) or \
```

Comment:
you can do `isinstance(X, (cudf.Series, cudf.DataFrame, pd.Series, pd.DataFrame))` to save a little typing and reading